libs/libc/modlib/modlib_verify.c

```diff
@@ -28,14 +28,18 @@
 #include <debug.h>
 #include <errno.h>
 
+#include <nuttx/arch.h>
 #include <nuttx/elf.h>
 #include <nuttx/lib/modlib.h>
 
 /****************************************************************************
  * Private Constant Data
  ****************************************************************************/
 
-static const char g_modmagic[EI_MAGIC_SIZE] = EI_MAGIC;
+static const char g_modmagic[EI_MAGIC_SIZE] =
+{
+    0x7f, 'E', 'L', 'F'
+};
```

Comment:
I'm not sure why this change is needed as it reverts https://github.com/apache/nuttx/pull/9710

Reply:
Good point, it shouldn't